Documentation/planning/arcade-powered-source-build/implementation-plan.md

```diff
@@ -5,30 +5,30 @@ To get each repo building with the new source-build 5.0 plan, [Arcade-Powered So
 > ![](img/implementation-plan-graph.png)
 > [source (img/implementation-plan-graph.dgml)](img/implementation-plan-graph.dgml)
 
-| Tier | Repo | Owner | (Stage 1)<br>Build from source - 5.0 | (Stage 2)<br>Patch Removal | (Stage 3)<br>Move build params to eng dir | (Stage 4)<br>Input Intermediate Packages Available | (Stage 5)<br>Official build | (Stage 6)<br>Source-build PR validation | (Stage 7)<br>Prebuilt check / enforcement |
-| --- | --- | --- | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
-| sbrp | Source-build-reference-packages | [Dan Seefeldt](https://github.com/dseefeld) | ✔️ | ✔️ | | | | | |
-| Tools | sourcelink | [Tomas Matousek](https://github.com/tmat) | ✔️ | 4 | | | | | |
-| Tools | arcade | [Mark Wilkie](https://github.com/markwilkie) | ✔️ | ⏱ 11 | | | | | |
-| 1 | application-insights | [Reiley Yang](https://github.com/reyang) | ✔️ | ✔️ | | | | | |
-| 1 | aspnet-xdt | [Vijay Ramakrishnan](https://github.com/vijayrkn) | ✔️ | 2 | | | | | |
-| 1 | newtonsoft-json | [Chris Rummel](https://github.com/crummel) | ✔️ | ✔️ | | | | | |
-| 1 | netcorecli-fsc | [Chris Rummel](https://github.com/crummel) | ✔️ | ✔️ | | | | | |
-| 1 | newtonsoft-json901 | [Chris Rummel](https://github.com/crummel) | ✔️ | ✔️ | | | | | |
-| 1 | xliff-tasks | [William Li](https://github.com/wli3) | ✔️ | 1 | | | | | |
-| 1 | clicommandlineparser | [Sarah Oslund](https://github.com/sfoslund) | ✔️ | 1 | | | | | |
-| 1 | roslyn | [Fred Silberberg](https://github.com/333fred) | ✔️ | 5 | | | | | |
-| 2 | linker | [Dan Seefeldt](https://github.com/dseefeld) | ✔️ | 3 | | | | | |
-| 2 | runtime | [Jared Parsons](https://github.com/jaredpar) | ✔️ | 15 | | | | | |
-| 2 | msbuild | [Ben Villalobos](https://github.com/BenVillalobos) | ✔️ | 4 | | | | | |
-| 2 | NuGet.Client | [Fernando Aguilar Reyes](https://github.com/dominoFire) | ✔️ | ⏱ 10 | | | | | |
-| 2 | templating | [Jose Aguilar](https://github.com/donJoseLuis) | ✔️ | ✔️ | | | | | |
-| 3 | aspnetcore | [John Luo](https://github.com/JunTaoLuo) | ✔️ | 13 | | | | | |
-| 3 | websdk | [Vijay Ramakrishnan](https://github.com/vijayrkn) | ✔️ | 1 | | | | | |
-| 4 | sdk | [Sarah Oslund](https://github.com/sfoslund) | ✔️ | 9 | | | | | |
-| 4 | vstest | [Jakub Jares](https://github.com/nohwnd) | ✔️ | 7 | | | | | |
-| 4 | fsharp | [Brett Forsgren](https://github.com/brettfo) | ✔️ | 4 | | | | | |
-| 5 | installer | [Sarah Oslund](https://github.com/sfoslund) | ✔️ | 3 | | | | | |
+| Tier | Repo | Owner | (Stage 1)<br>Build from source - 5.0 | (Stage 2)<br>Input intermediate packages available | (Stage 3)<br>Merge patches & local build infra into dev branch | (Stage 4)<br>Merge CI, generate mock official build | (Stage 5)<br>Validate & merge dev branch into 5.0 | (Stage 6)<br>Prebuilt baseline enforcement |
+| --- | --- | --- | :---: | :---: | :---: | :---: | :---: | :---: |
+| sbrp | Source-build-reference-packages | [Dan Seefeldt](https://github.com/dseefeld) | ✔️ | ✔️ | ✔️ | ✔️ | ✔️ | |
```

Comment:
This build infra doesn't check prebuilts yet, which is implied to be done in (3) because we want to have analysis running but not enforced yet at this point. Working on https://github.com/dotnet/source-build/issues/1715, tracking.

Reply:
I marked stage 3 as "in progress" and linked the issue to make that clear.